MessengerOnFileForLinux/Common/src/SignalHandling.cpp

```diff
@@ -82,4 +84,46 @@ void createPosixSignalsHandling(void(*handlingFunction)(int))
         std::signal(posixSignal, handlingFunction);
     }
 }
+
+namespace NCurses
+{
+void resizeHandlerInMainWindow(int)
+{
+    fileLog("Console resize handled in main window", LogSpace::Common);
```

Comment:
czy te fileLogi nie zajebią bufor logów jesli ktoś będzie ruszał? Czy tylko raz zarejestrują to?

Reply:
troche tak, ale log jest wazny według mnie

Reply:
Ale jak jest -> log jest gdy ktoś zmieni rozmiar okienka czy przy każdej literze czy czymśtam podczas zmiany okienka? to jest bardzo ważne.

Reply:
gdy ktoś zmieni rozmiar okienka

Reply:
może zostać